src/interfaces.ts

```diff
@@ -1,11 +1,16 @@
 export const RESEND_ERROR_CODES_BY_KEY = {
-  missing_required_fields: 422,
+  missing_required_field: 422,
+  invalid_access: 422,
+  invalid_parameter: 422,
```

Comment:
```suggestion
```
we don't actually return these errors from the API, we can remove them

Reply:
btw, here's an updated list with the errors we do throw, we can update them in the SDK as well:
```
| 'missing_required_parameter'
    | 'missing_required_field'
    | 'invalid_attachment'
    | 'missing_api_key'
    | 'restricted_api_key'
    | 'not_found'
    | 'rate_limit_exceeded'
    | 'daily_quota_exceeded'
    | 'application_error'
    | 'validation_error'
    | 'security_error';
```
    I'll update our Docs to make sure they are aligned in there as well 👍

Reply:
I'm fairly certain I received both in my resend logs during setup. Relates to an invalid from address and a non validated domain 🤔

Reply:
Fwiw invalid Parma definitely does appear in my resend logs from using a bad from address. Not sure if you can verify but I can certainly confirm it in my dashboard.

Reply:
```
{
    "statusCode": 422,
    "name": "invalid_parameter",
    "message": "Invalid `from` field. The email address needs to follow the `email@example.com` or `Name <email@example.com>` format"
}
```

Reply:
you're right! we do return this one 👍